sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ItemDeserializer.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation;
+
+import com.azure.cosmos.implementation.encryption.EncryptionProcessor;
+import com.azure.cosmos.implementation.encryption.api.DataEncryptionKeyProvider;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+
+public interface ItemDeserializer {
```

Comment:
>ItemDeserializer [](start = 17, length = 16)

ItemDeserializer vs core serialzer contracts

Reply:
this is internal class, not public contract. For now keeping it as is.